mzLib/Proteomics/PSM/PsmTsvHeader.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Proteomics.PSM
+{
+    public class PsmTsvHeader
+    {
```

Comment:
Do we want to break out a parent class TsvHeader for the omics project? Can put the scan and sequence information in there